test/frontend/components/coded_set_table_form_test.js

```diff
@@ -19,7 +19,7 @@ describe('CodedSetTableForm', () => {
   });
 
   it('should create the table', () => {
-    expect(component.find("tr").length).to.equal(5);
+    expect(component.find("tr").length).to.equal(4);
```

Comment:
Yes, I agree,  just didn't want to deal with that right refactoring that portion of things right now.